docker/dev/init.sql

```diff
@@ -1,7 +1,8 @@
+CREATE USER IF NOT EXISTS 'exercism' IDENTIFIED BY 'exercism';
+
 CREATE DATABASE IF NOT EXISTS `exercism_development`;
 ALTER DATABASE `exercism_development` CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
-/* Implicitly create the user with our first DB to avoid errors if they already exist */
-GRANT ALL PRIVILEGES ON `exercism_development`.* TO 'exercism' IDENTIFIED BY 'exercism';
+GRANT ALL PRIVILEGES ON `exercism_development`.* TO 'exercism';
```

Comment:
Why this change? Just not necessary any more?

Reply:
It didn't work on my dev-env. I was looking into way to fix it